test/spec/ol/dom/dom.test.js

```diff
@@ -116,34 +115,29 @@ describe('ol.dom', function() {
 
     it('sets transform origin for IE 9', function() {
       // save old user agent information
-      var originalIsIE = goog.userAgent.IE;
-      var originalIsVersionOrHigher = goog.userAgent.isVersionOrHigher;
+      var originalIsIE = ol.has.IE;
```

Comment:
No need to change it here since this is the way things were, but it would be more robust to cache the original value in a `beforeEach` and restore it in `afterEach`.  The way things are here, if there is an unhandled error, the value isn't restored.